Yurii_Telishevskyi/todo/src/app/todocomponent/todocomponent.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+
+
+@Component({
+  selector: 'app-todocomponent',
+  templateUrl: './todocomponent.component.html',
+  styleUrls: ['./todocomponent.component.scss']
+})
+export class TodocomponentComponent {
```

Comment:
You don't need to use in the name word "component". The component is an entity in Angular